csrc/options.h

```diff
@@ -21,6 +21,14 @@ namespace nvfuser {
 //! These can be set through the `NVFUSER_DUMP` environment variable
 //!
 enum class DebugDumpOption {
+  FunctionTrace, //!< Dump the function trace of selected internal function. The
+                 //!< function of interest needs to be instrumented with
+                 //!< DEBUG_PRINT_SCOPE and optionally RECORD_AND_RETURN before
```

Comment:
Nice comment! Can you also explain what happens when RECORD_AND_RETURN is missing? I guess the function return still shows up just without the return value?

Reply:
added